integration-tests/actions/automationv2/actions.go

```diff
@@ -157,20 +160,16 @@ func (a *AutomationTest) SetDockerEnv(env *test_env.CLClusterTestEnv) {
 }
 
 func (a *AutomationTest) DeployLINK() error {
-	linkToken, err := a.Deployer.DeployLinkTokenContract()
+	linkToken, err := contracts.DeployLinkTokenContract(a.Logger, a.ChainClient)
 	if err != nil {
 		return err
 	}
 	a.LinkToken = linkToken
-	err = a.ChainClient.WaitForEvents()
-	if err != nil {
-		return errors.Join(err, fmt.Errorf("failed waiting for link token contract to deploy"))
-	}
```

Comment:
just to confirm, with seth, we dont need to WaitForEvents, and after this DeployLINK() function is done, we have LINK liquidity?

unrelated to this PR, but if we dont want LINK liquidity on the chain, just dont DeployLINK?

Reply:
All automation tests by default deploys its own LINK token on chain and uses that for the test. However, some tests like load test also have the capability to use existing link token (like the official one on a testnet) for a test.

Reply:
@shileiwill no need to call `WaitForEvents()` anymore, Seth has fully synchronous API, so functions return only once transaction has been minted.

At least that's the behaviour you get, when contract interaction is wrapped in `seth.Decode()` function. By default it waits for tx minting and if it was reverted it decodes inputs/outputs/events and traces all calls.